src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,40 @@
 import React from 'react';
+import { FullPost } from '../../types/FullPost';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+interface Props {
+  post: FullPost,
+}
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    user,
+    body,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
+          {user && <UserInfo user={user} />}
+        </p>
```

Comment:
```suggestion
          {user && <UserInfo user={user} />}

```

Reply:
In my opinion, "<p>{' Posted by  '} </p>" has nothing to do with the userInfo component, since only information about the user is stored there

Reply:
but if you haven't user we still render `Posted by`, I believe it related to UserInfo component so if we don't have user we render nothing